exporter.go

```diff
@@ -32,11 +35,11 @@ type Exporter struct {
 	cherrs chan error
 }
 
-func NewExporter(address string) *Exporter {
+func NewExporter(address string, connectionTimeout time.Duration) *Exporter {
```

Comment:
This change is breaking the public API, what about making the connection timeout optional and add a setter method instead ?